docs/howto/openstack/nova/boot-image-volume.md

```diff
@@ -0,0 +1,395 @@
+# Converting boot-from-image servers to boot-from-volume
+
+You can freely move any server [between {{brand}}
+regions](move-server-between-regions), provided it uses
+boot-from-volume. Since such servers offer much more flexibility than
+those using boot-from-image, we recommend the former over the latter.
```

Comment:
This reads a little awkward to me. Maybe we can change this to:
```suggestion
regions](move-server-between-regions), provided it boots from a volume
(which we generally recommend since it affords more flexibility than booting from an image).
```